eng/Version.Details.xml

```diff
@@ -25,5 +25,29 @@
       <Uri>https://github.com/microsoft/testanywhere</Uri>
       <Sha>aa2fcc8616d988b234bc1d218465b20c56d0b82f</Sha>
     </Dependency>
+    <!-- Intermediate is necessary for source build. -->
+    <Dependency Name="Microsoft.SourceBuild.Intermediate.diagnostics" Version="9.0.0-preview.24566.1">
+      <Uri>https://github.com/dotnet/diagnostics</Uri>
+      <Sha>8c505ca6921b5f7e9b8acc234cc8f15035537ee4</Sha>
+      <SourceBuild RepoName="diagnostics" ManagedOnly="true" />
+    </Dependency>
+    <!-- Intermediate is necessary for source build. -->
+    <Dependency Name="Microsoft.SourceBuild.Intermediate.source-build-externals" Version="9.0.0-alpha.1.24175.1">
```

Comment:
What version(s) of the SDK is this going to flow into?  Given arcade and SBRP is a 10.0 version, I would expect this to be the same version.

Reply:
This will flow into .net sdk 10. But above Amaury mentions that he wants to downgrade the build to 9.

https://github.com/microsoft/testfx/pull/4295#discussion_r1878054720

Reply:
@MichaelSimons in past when we were on net8 and starting to move to net9 we got some "complaints" for community that it was painful and limiting their ability to contribute to be forced to be on alpha/beta/preview SDK. I know `build.cmd` installs local etc.